cypress/integration/examples/viewport.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+context('Viewport', () => {
```

Comment:
The test context provided does not match the task description. The test should focus on Book Store app functionality, not the viewport.